cmd/kubeadm/app/phases/addons/proxy/proxy.go

```diff
@@ -56,11 +60,52 @@ func EnsureProxyAddon(cfg *kubeadmapi.MasterConfiguration, client clientset.Inte
 		return err
 	}
 
-	proxyConfigMapBytes, err := kubeadmutil.ParseTemplate(KubeProxyConfigMap, struct{ MasterEndpoint string }{
-		// Fetch this value from the kubeconfig file
-		MasterEndpoint: masterEndpoint})
+	// Validate the kube-proxy bind address
+	bindAddress := cfg.KubeProxy.Config.BindAddress
```

Comment:
remove this, and install call `pkg/proxy/apis/kubeproxyconfig/validation.Validate(cfg.KubeProxy.Config)` from `cmd/kubeadm/app/apis/kubeadm/validation.ValidateMasterConfiguration()`